packages/trader/src/Modules/CFD/Components/cfd-account-card.tsx

```diff
@@ -255,11 +255,9 @@ const CFDAccountCard = ({
         return `${type.category}-${type.type}${region_string}${server_number}`;
     };
 
-    const handleClickSwitchAccount = (): void => {
-        if (toggleShouldShowRealAccountsList && toggleAccountsDialog) {
-            toggleShouldShowRealAccountsList(true);
-            toggleAccountsDialog(true);
-        }
+    const handleClickSwitchAccount: () => void = () => {
```

Comment:
Do we need this change?
Your update is changing the condition of checking for both arguments before proceeding but now it checks one by one.
Also as mentioned before, make sure you follow the objective of your PR which is to do TS migration and not any other change!

Reply:
@matin-binary, Because it was a suggestion of Bala. You can see it in the upper scope.
![Screenshot 2022-02-15 at 10 03 06](https://user-images.githubusercontent.com/88426229/154009815-2fecb236-bb5f-4c7b-8c54-24d1c3aa7b74.png)
a

Reply:
@balakrishna-binary is this needed?

Reply:
@matin-binary if condition is added recently in one of the TS conversions - https://github.com/binary-com/deriv-app/pull/4746/files#diff-ca42367ddc635b79247bcae59e3c2402fd1b0e74f81a2431e2d9620b24d76e8bR259.

I think the condition is added to avoid TS compile error. I suggested adding optional chaining to improve the code readability.

Reply:
Thanks for clarification @balakrishna-binary .
@maryiafrantsava-deriv This is a change of logic back to its original condition, by right, there should be a proper justification for the changes in each PR!

Reply:
@matin-binary, ok, I'll keep it in mind
@balakrishna-binary, thank you